src/apiActionSteps.ts

```diff
@@ -17,7 +17,7 @@ import { sendHttpRequest } from './apiHelpers';
  * @param key key of the remembered value
  */
 When(
-  'I send {text} request to {landing-url}{headers} and save response as {text}',
```

Comment:
I would suggest to wrap all values that were {text} previously to wrap into getValue function to extract memory value. That's give more flexibility in designing test cases.
and add such login into landingUrlType

Reply:
I thought about adding `getValue()` to the steps where `{text}` type was used but I decided to ignore it due to it's not needed here. You will not use value from the memory for http methods for example

Reply:
btw in `landingUrl` we have logic to get value from the memory if needed =)